File: JAVA/BaseballGameTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class BaseballGameTest {

  @Test
	public void testCalPoints() throws Exception {
      //Test 1
      int want = 30;
      String[] ops = {"5","2","C","D","+"};
      int got = BaseballGame.calPoints( ops );
      assertEquals(want, got);
      //Test 2
      want = 27;
      ops = new String[]{"5","-2","4","C","D","9","+","+"};
      got = BaseballGame.calPoints( ops );
      assertEquals(want, got);
	}

}

File: JS/BaseballGame.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const GAME = require('./BaseballGame.js')

TEST('Starting test...', (t) => {
    //Test 1
    t.assert(GAME.calPoints(['5','2','C','D','+']) === 30, "Expect: 5 2 C D + = 30")
    //Test 2
    t.assert(GAME.calPoints(["5","-2","4","C","D","9","+","+"]) === 27, "Expect: 5 -2 4 C D 9 + + = 27")

    t.end()
})
